### PR TITLE
build: fix webpack config issues to support build bundles for browser

### PR DESCRIPTION
**Pull request by yorkie**
Hey @jan-www @risenW, I updated the `webpack.config.js` to support build bundles for the browser, and now the browser testing via karma is working at Chrome Headless, though.

BTW, I recovered the index entry because we should make all features be working except for "rand/beta", a wasm module that is based on emscripten, that'll be replaced with [rustwasm](https://rustwasm.github.io/book/).

And I don't intend to enable CI browser testing in this PR, and this should not have any impact on the Node.js part, to run the browser building and testing, try the following command:

```sh
$ npm run build:browser && npm run test:browser
```

### CODE REVIEW

**Comment by yorkie** [comment]
The reason that not enabling CI browser testing now is that the build refactor is incomplete, I will create the Node.js bundle via Webpack in my next Pull Request, then we could unify the builds for Node.js and browser, also include the whole tests and benchmarks.

**Comment by jan-www** [comment]
LGTM

**Review by jan-www — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -41,14 +41,14 @@
   },
   "homepage": "https://github.com/imgcook/datacook#README",
   "devDependencies": {
-    "babel-loader": "^8.2.2",
```
[the remark this answers is not in the document]
Comment (yorkie) [reply package.json:44]: The CI [build](https://github.com/imgcook/datacook/runs/4298904399?check_suite_focus=true) seems to be failed, I'm checking now.

**Comment by yorkie** [comment]
To address the [issue](https://github.com/imgcook/datacook/pull/79#discussion_r755046960), I did enable the browser building and testing on our CI and it's working now, let's keep it.

**Review by jan-www — APPROVED** [review]

**Pull request merged**